src/handlers/oracles.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
    * responses: 201, 400, 401, 403, 404, 405, 406, 501, 503
    */
   post: async (request, h) => {
-    await oracle.createOracle(request.payload)
+    await oracle.createOracle(request.payload, request.headers, request.span)
```

Comment:
Normally this is where we do the initial span.audit with action EventSdk.AuditEventAction.start --- example --> https://github.com/mojaloop/ml-api-adapter/blob/master/src/api/transfers/handler.js

Reply:
Initial span.audit with action EventSdk.AuditEventAction.start is at src/domain/oracle/oracle.js createOracle function. Please let me know if needs to come up at the src/handlers/oracles.js level.

Reply:
Initial span.audit with action EventSdk.AuditEventAction.start is at src/domain/oracle/oracle.js createOracle function. Please let me know if needs to come up at the src/handlers/oracles.js level.

Reply:
Same as above.

Reply:
Code has been refactored to have all newly added event sdk code at handlers level.